README.md

```diff
@@ -22,14 +22,27 @@ Next, clone the sample repo and install the NPM.
 
 From your shell or command line:
 
-* `$ git clone git@github.com:AzureADSamples/B2C-WebApp-OpenIDConnect-NodeJS.git`
+* `$ git clone git@github.com:AzureADQuickStarts/B2C-WebApp-OpenIDConnect-NodeJS.git`
 * `$ npm install`
 
-### Step 5: Configure your server using config.js
+### Step 4: Configure your server
 
-**NOTE:** You may also pass the `issuer:` value if you wish to validate that as well.
+Provide the parameters listed in config.js as instructed.
```

Comment:
Is this where people are supposed to replace <tenant_name> and <signin_policy_name>? We should call out the places they need to replace, since it's easy to miss the ones in the destroy session url.